src/ut/utils_test.cpp

```diff
@@ -198,3 +219,527 @@ TEST_F(UtilsTest, ParseIPAddresses)
   EXPECT_FALSE(parse_ip_target("1.2.3.4:8888", ai.address));
 }
 
+TEST_F(UtilsTest, Split)
+{
```

Comment:
Also, I assume the only changed/new tests are the stopwatch ones and the IO ones, yes?

Reply:
Correct.